README.md

```diff
@@ -16,8 +16,45 @@ This guide aims to improve the way your team uses [jQuery](http://jquery.com/).
 ## Table of Contents
 
 * [About jQuery](#about-jquery)
-
+* [Avoid using `.click()`, `.change()`, et al as event handlers]
 
 ## About jQuery
 
 [jQuery](http://jquery.com/) is a utility library for easy DOM access & manipulation, event handling, Ajax and more. By using jQuery you can write consise and expressive code which works across modern and legacy browsers. jQuery has extensive tests, detailed documentation, a large active community and an ecosystem of plugins.
+
+## Avoid using `.click()`, `.change()`, et al as event handlers
+### Why?
+Using the .on() method instead can reduce memory usage and works on dynamically added elements.
+
+Events added with .on() can also be namespaced. This make code easier to read and when removing the event you are sure you are not accidentally removing other event handlers.
+
+### How?
+``` javascript
+// Bad
+$('.todo-item').click(function(event) {
+  // ...
+});
+
+// Good
+$('.todo-item').on('click', function() {
+  // ...
+});
+
+// Better
+$('.todo-list').on('click', '.todo-item', function(event) {
+  // ...
+});
+```
+
+Using namespaced event handlers
+``` javascript
+// Adding namespaced event
+$('.todo-item').on('click.todoItem', function(event) {
+  // ...
+});
+
+// Remove namespaced event
+$('.todo-item').off('click.todoItem');
+```
+
```

Comment:
When the guideline is available about [building a custom jQuery](https://github.com/jquery/jquery/tree/3.0.0-beta1#how-to-build-your-own-jquery) without event aliases, link to that?
And / or add instruction here: `grunt custom:-event/alias`?